TUM Campus App/TUMPickerController.swift

```diff
@@ -0,0 +1,83 @@
+//
+//  TUMPickerController.swift
+//  Campus
+//
+//  Created by Tim Gymnich on 28.06.18.
+//  Copyright © 2018 LS1 TUM. All rights reserved.
+//
+
+import UIKit
+
+protocol TUMPickerControllerDelegateProtocol: class {
+    func didSelectOption(_ element: DataElement)
+}
+
+protocol TUMPickerControllerDelegate: TUMPickerControllerDelegateProtocol {
+    associatedtype Element: DataElement
+    func didSelect(element: Element)
```

Comment:
Should this method have a more descriptive name, e.g. didSelectOption like above or didSelectPickerOption?

Reply:
i think the naming is alright. I think naming it element fits since the picker displays DataElements.